Hai Reveal: pirate weapon plot holes, A09 text walls, Teeneep continuity, and more player engagement (#7658)
endless-sky/endless-sky: data/hai/hai reveal 4 middle.txt

				`	"It sounds like something I can help with, but there's something I need from you first."`
					goto infotrade
				# TODO: Allow the player to NOT provide information here, and investigate some other way in the future.
			label infotrade
			`	She seems quite familiar with the practice of trading information, as she simply says, "What do you need?"`
				goto tribute
			label direct
			`	Teeneep and the other Hai seem shocked. After a surprised exclamation in Hai, she says, "What!?"`
			`	The Hai laugh for a while until they notice your face. "Oh, you're not joking," says Teeneep.`
			label tribute
			`	You explain about pirates having Hai weapons, and about a "trusted source" that told you about an entire warehouse of them.`
			`	"That's... worrying, but the Elders are pacifists to a fault, so I can't imagine them doing it. It is more likely that smugglers took it there, or ordinary pilots sold their used Hai gear. Do you have more evidence?"`
			`	After you describe what Sayari said about Hai pirate deals, Teeneep shakes her head and says, "Sayari would put a stop to any weapon deals. I'm sure it's something innocuous, like cash or basic supplies for their anarchist worlds.`

> 			`	"If you find solid proof that our government is arming pirates, I can, and will, stop them. There are extreme legal maneuvers civilians can use to force the Elder Council to change their mind.`

Reviewer: ```suggestion
			`	"If you find solid proof that our government is arming pirates, I can, and will, stop them. There are extreme legal maneuvers civilians can use to force the Elder Council to change their minds.`
```

	on offer
		log "People" "Teeneep Goot'r" `Teeneep is an activist and social leader among the Hai who heads an organization with hundreds of employees and many sponsors. She's used her extensive resources to search for answers about kidnapped specialists.`
		log `Many Hai and humans have gone missing over the past few decades. Many of the ones who have subsequently been rescued have had mild cadmium poisoning from some technology not of human or Hai origin. Teeneep, a Hai social leader, is using her network of information-gatherers to find out what's going on. Heading to the Quarg south of Hai space to begin the search.`

> 		log `Teeneep doubts the Hai government would arm pirates, but she claims to be able to shut down that operation, if she has solid evidence.`

Reviewer 2: ```suggestion
		log `Teeneep doubts the Hai government would arm pirates, but she claims to be able to shut down any such operation, if she has solid evidence.`
```

Leaves it more open to that doubt versus point at "'that' operation."

> 			`	"Greetings Captain <last>, my name is Teeneep Goot'r. I'm a social advocate among the Hai, and I'm trying to help your friends with a problem. As you know, they were kidnapped by humans. A few other Hai and human victims are here; they were mostly rescued by other kind merchants in human space, but around half of those here are my staff."`

Reviewer 3: If you're about to launch into asking her about the thing you're meant to be finding out about *discretely*, there should at least be some stated separation from others.

```suggestion
			`	"Greetings Captain <last>, my name is Teeneep Goot'r. I'm a social advocate among the Hai, and I'm trying to help your friends with a problem. As you know, they were kidnapped by humans. A few other Hai and human victims are here; they were mostly rescued by other kind merchants in human space, but around half of those here are my staff." As she speaks she directs you to some seats that are just far enough away from the other people in the room to confer at least the illusion of privacy.`
```

			`	"My organization has been researching the disappearance of Hai ever since humans began disappearing here under similar circumstances. We've discovered some patterns to these abductions, and through those we've been able to reach some of the victims more quickly upon their return, and even rescue some ourselves. But we have noticed something strange among the abductees: those Hai that you rescued, and several others, too, are all experiencing mild cadmium poisoning."`
			choice
				`	"Before we discuss that, can you tell me if the Hai government is arming pirates?"`
					goto direct
				`	"It sounds like something I can help with, but there's something I need from you first."`
					goto infotrade
			label infotrade
			`	She seems quite familiar with the practice of trading information, as she simply says, "What do you need?"`
				goto tribute
			label direct

> 			`	Teeneep and the other Hai seem shocked. After a surprised exclamation in Hai, she says, "What!?"`

Reviewer 3: Have we established a reason that you might be inclined to consider her your best avenue for asking up front?
Like she obviously is to us as the writers, but I think it feels perhaps a little rushed.
(If there's a lead-in justification somewhere else though feel free to point me at it, I may have overlooked it.)

Also, not sure if a surprised exclamation is appropriate here.
If you showed up and asked me something equivalently ridiculous (from her pov) like, "Is your boss selling hamsters to illegal pet traders?" my response would not be "What?!" and be more along the lines of "<concerned facial expression> ...are you high?"

Author: I have updated three missions, so the contact with Teeneep makes more sense. 

1. Sayari's note in the Mountaintop-furnishing mission vouches for Teeneep's organization and asks you to trade information and resources with her to solve the pirate troubles. (Her request is underscored by a substantial payment.) 
2. When you talk to Katya, she vouches for Sayari if you mention her by name, or is happy about a lead if you don't.
3. Before you ask Teeneep about the pirate arms deals, the narrator muses that this may be your only opportunity to pursue information, but warns you to be careful with your words. If you choose the direct approach, the Hai in the room laugh at you "for a while" until they see that you're serious.

>  If you showed up and asked me something equivalently ridiculous (from her pov) like, "Is your boss selling hamsters to illegal pet traders?" my response would not be "What?!" and be more along the lines of " ...are you high?"

Obviously, the problem is that you have a boring boss.

			`	"Greetings Captain <last>, my name is Teeneep Goot'r. I'm a social advocate among the Hai, and I'm trying to help your friends with a problem. As you know, they were kidnapped by humans. A few other Hai and human victims are here; they were mostly rescued by other kind merchants in human space, but around half of those here are my staff."`
			`	"My organization has been researching the issue of disappearing Hai ever since some humans also began disappearing here under similar circumstances, and we've discovered some patterns to the disappearances. Perhaps more pertinently though, since we've found those patterns we've been able to reach some of the victims more quickly upon their return and even rescue a couple ourselves. However, those Hai that you rescued and several others, too, have a very strange commonality. These kidnap victims are all experiencing mild cadmium poisoning."`
			choice
				`	"Before we discuss that, can you tell me if the Hai government is arming pirates?"`
					goto direct
				`	"It sounds like something I can help with, but there's something I need from you first."`
					goto infotrade
			label infotrade

> 			`	She sounds familiar with the information trade, as she says, "What do you need?"`

Reviewer 2: ```suggestion
			`	She seems quite familiar with the practice of trading information, as she simply says, "What do you need?"`
```

Another slight tone adjustment, a little less formal acknowledgment of the arrangement but also more assumptive about Teeneep's usage of it.

				goto tribute
			label direct
			`	Teeneep and the other Hai seem shocked. After a surprised exclamation in Hai, she says, "What!?"`
			label tribute
			`	You explain about pirates having Hai weapons, and about a "trusted source" that told you about an entire warehouse of them.`
			`	"That's... worrying, but the Elders are pacifists to a fault, so I can't imagine them doing it. It is more likely that smugglers took it there, or ordinary pilots sold their used Hai gear. Do you have more evidence?"`

> 			`	After you describe what Sayari said about Hai pirate deals, Teeneep shakes her head and says, "Sayari would put a stop to any weapon deals. I'm sure it's something innocuous, like cash, or basic supplies for their anarchist worlds.`

Reviewer 2: ```suggestion
			`	After you describe what Sayari said about Hai pirate deals, Teeneep shakes her head and says, "Sayari would put a stop to any weapon deals. I'm sure it's something innocuous, like cash or basic supplies for their anarchist worlds.`
```

				goto conclusion
			
			label conclusion

> 			`	"You must take me to <planet> immediately so I can do my due diligence and investigate this personally. I cannot land my own ship there for now because of... bureaucratic reasons. My people here can analyze any sensor logs or other data you have on Scar's Legion. You've found solid proof of the Elders arming pirates, though, I doubt this is a lead on the kidnappings. At <planet>, I can catch up with our mutual friend from the secure spaceport. I have sufficient privileges to be there, even if I have no clearance for landing my ship."`

Reviewer: ```suggestion
			`	"You must take me to <planet> immediately so I can do my due diligence and investigate this personally. I cannot land my own ship there for now because of... bureaucratic reasons. My people here can analyze any sensor logs or other data you have on Scar's Legion. You've found solid proof of the Elders arming pirates, though I doubt this is a lead on the kidnappings. At <planet>, I can catch up with our mutual friend from the secure spaceport. I have sufficient privileges to be there, even if I have no clearance for landing my ship."`
```

				goto conclusion
			
			label conclusion

> 			`	"You must take me to <planet> immediately so I can do my due diligence and investigate this personally. I cannot land my own ship there for now because of... bureaucratic reasons. My people here can analyze any sensor logs or other data you have on Scar's Legion. You've found solid proof of the Elders arming pirates, though I doubt this is a lead on the kidnappings. At <planet>, I can catch up with our mutual friend from the secure spaceport. I have sufficient privileges to be there, even if I have no clearance for landing my ship."`

Reviewer 3: The way this is laid out feels clunky to me so I propose an alternative arrangement.
```suggestion
			`	"You must take me to <planet> immediately. This is something I should investigate this personally if I hope to find the truth of things and I cannot land my own ship there right now thanks to... bureaucratic reasons. My people here can analyze any sensor logs or other data you have on Scar's Legion. You've found solid proof of the Elders arming pirates, though I doubt this is a lead on the kidnappings. At <planet>, I will catch up with our mutual friend from the secure spaceport. I have sufficient privileges to be there, even if I have no clearance for landing my ship."`
```

			`	She waves over Wayou towards you. "Please, give Wayou any relevant data files while I pack for departure. I suspect that I have kept you away from <planet> far too long as it is. According to my sources, people there await your return."`
			`	You've barely had time to load your logs onto a data stick and hand them to Wayou before Teeneep is packed and waiting to depart. It would seem your course is set.`
				accept

> mission "Pirates with Hai Weapons 2: Await Teeneep's Research"

Reviewer 3: Is this mission necessary?

Author: The "Pirates with Hai Weapons" mission description looks like the plot forgot about it if I don't update it here.

				goto conclusion
			
			label conclusion

> 			`	"You must take me to <planet> immediately. This is something I should investigate this personally if I hope to find the truth of things and I cannot land my own ship there right now thanks to... bureaucratic reasons. My people here can analyze any sensor logs or other data you have on Scar's Legion. You've found solid proof of the Elders arming pirates, though I doubt this is a lead on the kidnappings. At <planet>, I will catch up with our mutual friend from the secure spaceport. I have sufficient privileges to be there, even if I have no clearance for landing my ship."`

Reviewer: ```suggestion
			`	"You must take me to <planet> immediately. This is something I should investigate this personally if I hope to find the truth of things, but I cannot land my own ship there right now because of... bureaucratic reasons. My people here can analyze any sensor logs or other data you have on Scar's Legion. You've found solid proof of the Elders arming pirates, though I doubt this is a lead on the kidnappings. At <planet>, I will catch up with our mutual friend from the secure spaceport. I have sufficient privileges to be there, even if I have no clearance for landing my ship."`
```
I think phrases like "thanks to" are really overdone in ES but especially in this campaign.

	on visit
		dialog `Teeneep is on another ship not yet in the system, take off and wait for the ship to arrive in order to continue.`
	on offer
		log `A group of Hai Elders agreed to change their approach to the problem of pirates if the situation deteriorates further. Unfortunately, they refused to even discuss the disappearances.`
		"reputation: Hai (Unfettered)" >?= 10
		conversation

> 			`You return, giving Batu directions to Xilin, and check availabilities. The other diplomats all appear to be busy and so you soon find yourself at Sayari's office only to be met by Teeneep and Sayari arguing. Teeneep is shouting angrily at Sayari who is trying to calm her. Their argument is so intense that they don't hear you approaching.`

Reviewer 2: ```suggestion
			`As you return, you give Batu directions to find Xilin and then check availabilities. The other diplomats all appear to be busy, so you soon find your own way to Sayari's office, only to be met by the sight of Teeneep and Sayari arguing. Teeneep is shouting angrily at Sayari, who is trying to calm her. Their argument is so intense that they don't hear you approaching.`
```

That first sentence is still a bit awkward; I can't really parse out what specifically is meant by "check availabilities." I'm guessing "of the other diplomats," but strictly because of the proximity in which they are the next subject. I'm not seeing any other connection in context from the last mission -or- relevance to the rest of the paragraph that explains it. If I'm correct about that, I think "and then check availabilities" can be stricken entirely and it will only improve this whole bit.

Author: I left in "availabilities" because I only 70% understood what the original writer was going for; I think it was trying to sound businessy, like "NDAs" in another mission, but that makes less sense coming from the narrator, rather than a businessperson. I'll rewrite the first paragraph.

Reviewer 2: Once that's sorted, I think this section is all good to go as well.

> 			`Upon landing, you give Batu directions to Xilin, and arrange meetings with diplomats in the evening. To pass the time, you search for Teeneep and Sayari, hoping to swap information with the two. You find yourself at Sayari's office only to be met by Teeneep and Sayari arguing. Teeneep is shouting angrily at Sayari who is trying to calm her. Their argument is so intense that they don't hear you approaching.`

Reviewer 2: ```suggestion
			`Upon landing, you give Batu directions to Xilin and arrange meetings with diplomats in the evening. To pass the time, you search for Teeneep and Sayari, hoping to swap information with the two. You find yourself at Sayari's office only to be met by Teeneep and Sayari arguing. Teeneep is shouting angrily at Sayari who is trying to calm her. Their argument is so intense that they don't hear you approaching.`
```

I think I missed this first time around with how awkward it was.

			label questions
			choice

> 				# First phase of conversation: interrupt the women.

Reviewer 2: ```suggestion
				# First phase of conversation: listen in or interrupt.
```

Just matching these up since we can, make it that much easier to sort it out when referencing.

						has "Hai Reveal [A09] History of Appeasement: label deals"
						not "Hai Reveal [A09] History of Appeasement: label fix"
				# Optional follow-up questions are next, so you'll get the critical
				# information first.
				`	"Are the two of you usually this argumentative?"`
					goto friends
					to display
						has "Hai Reveal [A09] History of Appeasement: interrupted"
						not "Hai Reveal [A09] History of Appeasement: label friends"
				`	"Did all of the deals involve arming pirates?"`
					goto types
					to display
						has "Hai Reveal [A09] History of Appeasement: label deals"
						not "Hai Reveal [A09] History of Appeasement: label types"
				`	"Does Hai media not deal with this sort of thing?"`
					goto media
					to display
						has "Hai Reveal [A09] History of Appeasement: label fix"
						not "Hai Reveal [A09] History of Appeasement: label media"
				# End discussion.
				`	"We've discussed enough. Let's get to work."`
					goto go
					to display
						has "Hai Reveal [A09] History of Appeasement: label fix"

			# First phase of conversation: listen in or interrupt.
			label listen
			action
				set "Hai Reveal [A09] History of Appeasement: label listen"

> 			`	The conversation is entirely in Hai, and you struggle to understand. At first, Teeneep is annoyed, maybe saying words like: pirate, victim, orchestra, and betrayal. Sayari's responses revolve around: necessity, an angry child, guilt, and eggs. Something sets off Teeneep, and she keeps screaming "tweezers." Sayari puts a paw gently on Teeneep's shoulders and says in a soothing maternal tone, "green the moon blue." At this point, you realize you don't speak Hai, and have no idea what the two women are discussing.`

Reviewer 2: ```suggestion
			`	It is difficult to hear precisely without making yourself known, so you struggle to understand. At first, Teeneep is annoyed, maybe saying words like: pirate, victim, orchestra, and betrayal. Sayari's responses revolve around: necessity, an angry child, guilt, and eggs. Something sets off Teeneep, and she keeps screaming "tweezers." Sayari puts a paw gently on Teeneep's shoulders and says in a soothing maternal tone, "green the moon blue." At this point, you realize the conversation is entirely in Hai, which you don't speak, and you have no idea what the two women are actually discussing.`
```

Let's bury that lead down into the punchline.

				goto questions
			label childish
			action
				set "Hai Reveal [A09] History of Appeasement: interrupted"
			`	As the pair turn towards you, Teeneep looks mortified. "Captain <last>, I... I'm sorry you had to see me like this. I just... Sayari?"`
			`	"Welcome back, Captain <last>. I must apologize for my friend's state. She's just discovered some rather unfortunate information."`
				goto arrived
			label interrupt
			action
				set "Hai Reveal [A09] History of Appeasement: interrupted"
			`	Realizing that you've arrived, Sayari tilts her head slightly to one side to indicate she's speaking to you, while not taking her eyes off Teeneep. "Welcome back, Captain <last>. I must apologize for my friend's state. She's just discovered some rather unfortunate information."`
				goto arrived
			label arrived
			`	"Unfortunate!" Teeneep very nearly shrieks the word. "They've been doing this for decades! Decades, Sayari! Tracking down and ultimately paying off pirates with our technology to leave us alone, even when they've hurt, killed, kidnapped, and enslaved our people!`
			`	"You knew! This whole time, for a half century, you knew, and you did nothing!"`

> 			`	Teeneep closes her eyes and uses a deep breathing technique to calm herself down. It looks like she's practiced this many times.`

Reviewer 2: ```suggestion
			`	Teeneep closes her eyes and breathes deeply until she has calmed herself down. It looks like this is a technique she's practiced many times.`
```

Chances are we're not necessarily aware it's a specific deep breathing technique, even though that would be a technically correct description of it in hindsight.

			`	The three of you sit down once Teeneep looks ready to talk.`
				goto questions
			label friends
			action
				set "Hai Reveal [A09] History of Appeasement: label friends"
			`	Sayari looks at you apologetically. "She's always been quite a spark, but we usually get along very well."`
			`	Teeneep adds, "Sayari has been my close friend and mentor since I was a head shorter than her."`
				goto questions

			# Second phase of conversation: find out about the pirates.
			label deals
			action
				set "Hai Reveal [A09] History of Appeasement: label deals"

> 			`	Sayari says, "The Elders have been making deals with pirates for a long time, in return for secrecy or ceasing mischef. We give them credits, supplies for their colonies, or weapons. I'm not privy to the details, but I've know for quite a while."`

Reviewer 2: ```suggestion
			`	Sayari says, "The Elders have been making deals with pirates for a long time, in return for secrecy or ceasing mischief. We give them credits, supplies for their colonies, or weapons. I'm not privy to the details, but I've know for quite a while."`
```

> 			`	An enraged Teeneep counters. "They gave them our weapons so that they could more efficiently go and enslave other people rather than just dealing with them properly they way they should have! One hundred and seventy three reports, Sayari. That's how many times they've done this, or something like it, and those are just the ones that came up in the sweep. There's no telling how many others might be filed improperly, missing key words, or just outside of my search range.`

Reviewer 2: ```suggestion
			`	An enraged Teeneep counters. "They give them our weapons so that they can more efficiently go and enslave other people, rather than just dealing with them properly the way they should have! One hundred and seventy three reports, Sayari. That's how many times they've done this, or something like it, and those are just the ones that came up in the sweep. There's no telling how many others might be filed improperly, missing key words, or just outside of my search range.`
```

Tense match.

> 			`	"I thought you were a model of morals, Sayari. Now I know you're no better than they are. No better than... than..."`

Reviewer 2: ```suggestion
			`	"I thought you were a model of morality, Sayari. Now I know you're no better than they are. No better than... than..."`
```

This one could remain the same if the intent is for it to be part of translated speech patterns or the like, but IIRC, technically morality is more correct to reference someone modeling.

			`	Sayari finishes Teeneep's thought. "No better than you? The curse of power is to sacrifice your morals for protection of people under your care. I think you know that well, Teeneep. Perhaps better than I?" Teeneep looks briefly ashamed when Sayari says that.`
				goto questions
			label types
			action
				set "Hai Reveal [A09] History of Appeasement: label types"

> 			`	Teeneep says, "In the cases I found, a third of the deals included weapons: shield regenerators, guns, turrets, and anti-missile systems. Most of the deals included something else too, like money, non-hostile technology, or supplies for anarchist worlds. For the past five years though, nearly every deal had weapons in the trade." She looks irate at Sayari for a second.`

Reviewer 2: ```suggestion
			`	Teeneep says, "In the cases I found, a third of the deals included hostile technology: shield regenerators, guns, turrets, or anti-missile systems. Most of the deals included something else too, like money, non-hostile technology, or supplies for anarchist worlds. For the past five years though, nearly every deal included weapons in the trade." She looks irate at Sayari for a second.`
```

When she already directly references "non-hostile" right after that, let's not call shield generators "weapons" ;) Also, switch to "or" -unless- it's meant to imply 1/3 of the deals always involved each of those hostile technologies.

"Include" is just more specific than "had."

				goto questions

			# Third phase of the conversation: decide to fix the problem.
			label fix
			action
				set "Hai Reveal [A09] History of Appeasement: label fix"
			`	Sayari explains. "The grievance is against the Elders. There's a process to seek redress of grievances with the Elders. Teeneep needs to at least attempt to follow that process, and if she does not go through with normal resolution channels, she will be censured from public life. She must confront the Elders directly."`

> 			`	Teeneep makes the effort to give a final, defiant snarl of disgust. "She's right," she grates out. "I hate it but she's right. Especially because several of those reports indicate that the Navy knew of Hai on some ships and stood by while they were taken away. Which makes it your business too."`

Reviewer 2: ```suggestion
			`	Teeneep makes the effort to give a final, defiant snarl of disgust. "She's right," she grates out. "I hate it, but she's right. Especially because several of those reports indicate that the Navy knew of Hai on some ships and stood by while they were taken away. Which makes it your business, too."`
```

> 			`	Thinking back to your meeting with Katya, you realize this is step closer to a "hostile diplomatic incident."`

Reviewer 2: ```suggestion
			`	Thinking back to your meeting with Katya, you realize this is probably a step closer to a "hostile diplomatic incident."`
```

"A" just missing, but it begs for a descriptor in the sentence, anyway. "Possibly" works just as well. Could also go with "definitely" or some such instead of you prefer that tone on it, though.

				goto questions
			label media
			action
				set "Hai Reveal [A09] History of Appeasement: label media"
			`	Sayari gives you a quizzical look, "Of course they do, though Tee has access to more efficient options. That's not the point, though."`
				goto questions
			label go

> 			`	Sayari nods at you and looks toward Teeneep. "Look, the Elders are pretty much permanently on call at the moment. None of them are leaving <origin> and they're in session as we speak. Let me put a priority audience request into the system, and you can probably see them this afternoon. In the meantime, why don't I order some food here and you can use my spare desk over there to prepare what you need to say?"`

Reviewer 2: ```suggestion
			`	Sayari nods at you and looks toward Teeneep. "Look, the Elders are pretty much permanently on call at the moment. None of them are leaving <origin>, and they're in session as we speak. Let me put a priority audience request into the system, and you can probably see them this afternoon. In the meantime, why don't I order some food here? You can use my spare desk over there to prepare what you need to say."`
```

Last was also missing a comma, but breaking it up instead cuts down the repetitiveness. Just fixing it leaves three "X, and X" sentences in a row.

> 			`	Teeneep takes a deep breath and accepts the plan with sudden and surprising grace. She gathers her pads and sets up where indicated. "Alright, just get me some comfort food while I work on this."`

Reviewer 2: ```suggestion
			`	Teeneep takes a deep breath and accepts the plan with sudden and surprising grace. She gathers her pads, then sets up where indicated. "Alright, just get me some comfort food while I work on this."`
```

Repetition swap again.

			`	Sayari asks you, "Her favorite foods are acorn curry and nutmeg pie. Which should we order?"`
			choice
				`	"Acorn curry sounds... good?"`
					goto curry
				`	"I'd love some nutmeg pie."`

> 			`	The pie tastes and feels like a meaty shepherd's pie, even though it's all nuts and vegetables. The subtle taste of nutmeg really brings it all together, and accents the acorns quite well.`

Reviewer 2: ```suggestion
			`	The pie tastes and feels like a meaty shepherd's pie, even though it's all nuts and vegetables. The subtle taste of nutmeg really brings it all together, accenting the acorns quite well.`
```

				goto meeting
			label curry
			`	You eat the most delicious curry you've ever tasted, with a perfect combination of acorns and the best Indian spices. It's amazing what Hai can do with nuts; no wonder Teeneep considers this a comfort food.`

			# Scene change: the meeting with the Elders begins.
			label meeting
			`	After a short hour of frantic work, Teeneep leads you to a room with three Hai Elders. Sayari explains, "This is a Special Adjunctive Triad: three Elders with diverse views, specialized knowledge, and the power to take limited immediate actions."`
			`	"Respected Elders," Teeneep begins, "I have extensive evidence of a sustained kidnapping campaign in Hai space, facilitated by pirates you've been arm-"`

> 			`	"Enough," says an Elder, raising his hand. "We know about your views, and the 'five thousand missing Hai,' and your organization's media connections. You're already risking larger issues with your current actions. We're here to speak to your witness. Captain <last>," he says, turning to you, "tell us your views on these matters."`

Reviewer 2: ```suggestion
			`	"Enough," says an Elder, raising his hand. "We know about your views, the 'five thousand missing Hai,' and your organization's media connections. You're already risking larger issues with your current actions. We're here to speak to your witness. Captain <last>," he says, turning to you, "tell us your views on these matters."`
```

Another technical correction that could be left if the intent is for the speech to be incorrect. Highlighting this one because I do personally think it's very awkward to see in writing, but -many- people do indeed speak this way.

			`	The left Elder says, "Secrecy, mostly, to prevent a refugee crisis if the Hai become common knowledge. The bulk of the resources go to a few large pirate groups that control the anarchist worlds. We fund the development and protection of their planets, in return for them ensuring the wrong people don't find us."`
			`	Another Elder adds, "Most of the deals are much smaller, though: when a pirate gang causes mischief, we pay them to stop. Hostages, theft, raids, and the like. Usually, the same gang never bothers us again." Teeneep perks up at the word "hostages," but doesn't speak.`
				goto "elder questions"
			label "elders secret"
			`	The middle Elder counters, "We cannot be certain that the pirate activity will continue. With the proper trades, we may be able to return to the status quo."`
			`	"That is a best-case scenario," says the right Elder, "but if matters continue to worsen, we'll change our approach." The other two nod in agreement.`
				goto diplomats
			label "Katya warning"
			`	The Elder on the right disagrees. "At this point the Republic is to weakened to be a threat. I'm confident they will see reason, for now."`

> 			`	Fortunately, the middle Elder contradicts him. "We don't want to risk larger problems with humanity. If the situation continues to worsen, then we'll change our approach." The three find that argument convincing and nod.`

Reviewer 2: ```suggestion
			`	Fortunately, the middle Elder contradicts him. "We don't want to risk larger problems with humanity. If the situation continues to worsen, then we'll change our approach." The other two find that argument convincing and nod.`
```

If this was meant to be all three of them becoming convinced and nodding, including the one that just spoke, the inclusion of the one that just spoke should be more directly/clearly noted.

			label diplomats
			`	An Elder concludes the meeting. "We have a plan of action on the pirate deals. However, I've heard enough of the 'kidnapping' conspiracy theory, so that ends this meeting." Teeneep is visibly furious, but remains silent as the three exit.`
			choice
				`	"That could have gone better."`
				`	"It sounds like we've made some progress!"`
			`	As you walk back to diplomatic areas, Sayari begins. "They didn't do anything, but at least they agreed to solve the problems when they become disasters."`
			`	Teeneep objects, "We can't wait for a disaster before solving this!"`

> 			`	Sayari agrees. "You're right, of course, in this matter. The Elders have ever struggled with change. They're not wise for this, they were not elected to be so. They were elected to continue the long march of incremental development. Besides, you know far better than most exactly what challenges they contend with these past couple decades. More of them have died or retired than been unelected."`

Reviewer 2: ```suggestion
			`	Sayari agrees. "You're right, of course, in this matter. The Elders have ever struggled with change. They're not wise for this, but they were not elected to be so. They were elected to continue the long march of incremental development. Besides, you know far better than most exactly what challenges they contend with these past couple decades. More of them have died or retired than been unelected."`
```

			choice
				`	"What can we do to solve this, Sayari?"`
				`	"Teeneep, I think we need quick action. You mentioned something back in the cave."`

> 			`	Before she can respond, Xilin arrives with Batu in tow. They consult Sayari (the actual diplomat) and occasionally Teeneep. After a long dialog about the Navy knowing of the pirate troubles and willfully not intervening. Xilin sympathizes, suggesting the Hai ask the Navy to rescue those captured.`

Reviewer 2: ```suggestion
			`	Before she can respond, Xilin arrives with Batu in tow. They consult Sayari (the actual diplomat) and occasionally Teeneep. After a long dialog about the Navy knowing of the pirate troubles and willfully not intervening, Xilin sympathizes, suggesting the Hai ask the Navy to rescue those captured.`
```

				goto arrived
			label arrived
			`	"Unfortunate!" Teeneep very nearly shrieks the word. "They've been doing this for decades! Decades, Sayari! Tracking down and ultimately paying off pirates with our technology to leave us alone, even when they've hurt, killed, kidnapped, and enslaved our people!`
			`	"You knew! This whole time, for a half century, you knew, and you did nothing!"`
			`	Teeneep closes her eyes and breathes deeply until she has calmed herself down. It looks like this is a technique she's practiced many times.`
			`	The three of you sit down once Teeneep looks ready to talk.`
				goto questions
			label friends
			action
				set "Hai Reveal [A09] History of Appeasement: label friends"
			`	Sayari looks at you apologetically. "She's always been quite a spark, but we usually get along very well."`
			`	Teeneep adds, "Sayari has been my close friend and mentor since I was a head shorter than her."`
				goto questions

			# Second phase of conversation: find out about the pirates.
			label deals
			action
				set "Hai Reveal [A09] History of Appeasement: label deals"

> 			`	Sayari says, "The Elders have been making deals with pirates for a long time, in return for secrecy or ceasing mischief. We give them credits, supplies for their colonies, or weapons. I'm not privy to the details, but I've know for quite a while."`

Reviewer: ```suggestion
			`	Sayari says, "The Elders have been making deals with your pirates for a long time, in return for secrecy or ceasing mischief. We give them credits, supplies for their colonies, or weapons. I'm not privy to the details, but I've known for quite a while."`
```

			`	An enraged Teeneep counters. "They give them our weapons so that they can more efficiently go and enslave other people, rather than just dealing with them properly the way they should have! One hundred and seventy three reports, Sayari. That's how many times they've done this, or something like it, and those are just the ones that came up in the sweep. There's no telling how many others might be filed improperly, missing key words, or just outside of my search range.`
			`	"I thought you were a model of morality, Sayari. Now I know you're no better than they are. No better than... than..."`
			`	Sayari finishes Teeneep's thought. "No better than you? The curse of power is to sacrifice your morals for protection of people under your care. I think you know that well, Teeneep. Perhaps better than I?" Teeneep looks briefly ashamed when Sayari says that.`
				goto questions
			label types
			action
				set "Hai Reveal [A09] History of Appeasement: label types"

> 			`	Teeneep says, "In the cases I found, a third of the deals included hostile technology: shield regenerators, guns, turrets, or anti-missile systems. Most of the deals included something else too, like money, non-hostile technology, or supplies for anarchist worlds. For the past five years though, nearly every deal included weapons in the trade." She looks irate at Sayari for a second.`

Reviewer: ```suggestion
			`	Teeneep says, "In the cases I found, a third of the deals included hostile technology: shield regenerators, guns, turrets, or anti-missile systems. Most of the deals included something else too, like money, non-hostile technology, or supplies for anarchist worlds. For the past five years, though, nearly every deal involved exchanging Hai weaponry." She looks irate at Sayari for a second.`
```

				goto questions

			# Third phase of the conversation: decide to fix the problem.
			label fix
			action
				set "Hai Reveal [A09] History of Appeasement: label fix"
			`	Sayari explains. "The grievance is against the Elders. There's a process to seek redress of grievances with the Elders. Teeneep needs to at least attempt to follow that process, and if she does not go through with normal resolution channels, she will be censured from public life. She must confront the Elders directly."`
			`	Teeneep makes the effort to give a final, defiant snarl of disgust. "She's right," she grates out. "I hate it, but she's right. Especially because several of those reports indicate that the Navy knew of Hai on some ships and stood by while they were taken away. Which makes it your business, too."`
			`	Thinking back to your meeting with Katya, you realize this is probably a step closer to a "hostile diplomatic incident."`
				goto questions
			label media
			action
				set "Hai Reveal [A09] History of Appeasement: label media"

> 			`	Sayari gives you a quizzical look, "Of course they do, though Tee has access to more efficient options. That's not the point, though."`

Reviewer: ```suggestion
			`	Sayari gives you a quizzical look. "Of course they do, though Tee has access to more efficient options. That's not the point, though."`
```

			`	Teeneep takes a deep breath and accepts the plan with sudden and surprising grace. She gathers her pads, then sets up where indicated. "Alright, just get me some comfort food while I work on this."`
			`	Sayari asks you, "Her favorite foods are acorn curry and nutmeg pie. Which should we order?"`
			choice
				`	"Acorn curry sounds... good?"`
					goto curry
				`	"I'd love some nutmeg pie."`
			`	The pie tastes and feels like a meaty shepherd's pie, even though it's all nuts and vegetables. The subtle taste of nutmeg really brings it all together, accenting the acorns quite well.`
				goto meeting
			label curry
			`	You eat the most delicious curry you've ever tasted, with a perfect combination of acorns and the best Indian spices. It's amazing what Hai can do with nuts; no wonder Teeneep considers this a comfort food.`

			# Scene change: the meeting with the Elders begins.
			label meeting

> 			`	After a short hour of frantic work, Teeneep leads you to a room with three Hai Elders. Sayari explains, "This is a Special Adjunctive Triad: three Elders with diverse views, specialized knowledge, and the power to take limited immediate actions."`

Reviewer: ```suggestion
			`	After a short hour of frantic work, Teeneep leads you to a small room with three Hai Elders. Sayari explains, "This is a Special Adjunctive Triad: three Elders with diverse views, specialized knowledge, and the power to take limited immediate actions."`
```
I want to do a bit more showing, and I'm going to have to get a feeling of what the room looks like.

> 			`	After a short hour of frantic work, Teeneep leads you to a room with three Hai Elders. Sayari explains, "This is a Special Adjunctive Triad: three Elders with diverse views, specialized knowledge, and the power to take limited immediate actions."`
> 			`	"Respected Elders," Teeneep begins, "I have extensive evidence of a sustained kidnapping campaign in Hai space, facilitated by pirates you've been arm-"`
> 			`	"Enough," says an Elder, raising his hand. "We know about your views, the 'five thousand missing Hai,' and your organization's media connections. You're already risking larger issues with your current actions. We're here to speak to your witness. Captain <last>," he says, turning to you, "tell us your views on these matters."`

Reviewer: ```suggestion
			`	After a short hour of frantic work, Teeneep leads you to a room within the government complex. Sayari explains, "This is a Special Adjunctive Triad: three Elders with diverse views, specialized knowledge, and the power to take limited immediate actions."` 
			`	The three Hai Elders are sat behind a table, and the Elder in the center gestures for all of you to sit down in those strange chairs. After a small, calm exchange in Hai between Sayari and the central Elder, he motions for you to speak.`
			`	Instead, Teeneep begins. "Respected Elders, I have extensive evidence of a sustained kidnapping campaign in Hai space, facilitated by pirates you've been arm-"`
			`	"Enough," says an Elder, raising his hand. "We know about your views, the 'five thousand missing Hai,' and your organization's media connections. You're already risking larger issues with your current actions. We are here to speak to your witness. Captain <last>," he says, turning to you, "tell us your views on these matters."`
```
I want to do a bit more showing, and I'm going to have to get a feeling of what the room looks like. I'm not so sure about how the Elders work or what impression they're trying to make, but I'm going to go from my gut here. I also suppose these small meetings are different in some ways to the big ones the player had been involved with before.

					goto "Katya warning"
			label "elders why"
			action
				set "Hai Reveal [A09] History of Appeasement: label elders why"
			`	The left Elder says, "Secrecy, mostly, to prevent a refugee crisis if the Hai become common knowledge. The bulk of the resources go to a few large pirate groups that control the anarchist worlds. We fund the development and protection of their planets, in return for them ensuring the wrong people don't find us."`
			`	Another Elder adds, "Most of the deals are much smaller, though: when a pirate gang causes mischief, we pay them to stop. Hostages, theft, raids, and the like. Usually, the same gang never bothers us again." Teeneep perks up at the word "hostages," but doesn't speak.`
				goto "elder questions"
			label "elders secret"
			`	The middle Elder counters, "We cannot be certain that the pirate activity will continue. With the proper trades, we may be able to return to the status quo."`
			`	"That is a best-case scenario," says the right Elder, "but if matters continue to worsen, we'll change our approach." The other two nod in agreement.`
				goto diplomats
			label "Katya warning"

> 			`	The Elder on the right disagrees. "At this point the Republic is to weakened to be a threat. I'm confident they will see reason, for now."`

Reviewer: ```suggestion
			`	The Elder on the right disagrees. "At this point, the Republic is too weakened to be a threat. I'm confident they will see reason, for now."`
```

			`	Fortunately, the middle Elder contradicts him. "We don't want to risk larger problems with humanity. If the situation continues to worsen, then we'll change our approach." The other two find that argument convincing and nod.`
			label diplomats

> 			`	An Elder concludes the meeting. "We have a plan of action on the pirate deals. However, I've heard enough of the 'kidnapping' conspiracy theory, so that ends this meeting." Teeneep is visibly furious, but remains silent as the three exit.`

Reviewer: ```suggestion
			`	An Elder concludes the meeting. "We have a plan of action on the pirate deals. However, I've heard enough of this 'kidnapping' conspiracy theory. Our meeting is adjourned." Teeneep's fury is palpable; she is tightly clutching the table, and appears to be ready to throw it, but she remains silent as the three exit.`
```

Author: This is waaaay more extreme of a reaction than I was going for. She's supposed to be largely under control by now; any strong emotions should not be obvious anymore.

Reviewer: Fair enough.

Reviewer: ```suggestion
			`	An Elder concludes the meeting. "We have a plan of action on the pirate deals. However, I've heard enough of this 'kidnapping' conspiracy theory. Our meeting is adjourned." Teeneep's fury is palpable, but she remains silent as the three exit.`
```

			`	Teeneep takes a deep breath and accepts the plan with sudden and surprising grace. She gathers her pads, then sets up where indicated. "Alright, just get me some comfort food while I work on this."`
			`	Sayari asks you, "Her favorite foods are acorn curry and nutmeg pie. Which should we order?"`
			choice
				`	"Acorn curry sounds... good?"`
					goto curry
				`	"I'd love some nutmeg pie."`
			`	The pie tastes and feels like a meaty shepherd's pie, even though it's all nuts and vegetables. The subtle taste of nutmeg really brings it all together, accenting the acorns quite well.`
				goto meeting
			label curry
			`	You eat the most delicious curry you've ever tasted, with a perfect combination of acorns and the best Indian spices. It's amazing what Hai can do with nuts; no wonder Teeneep considers this a comfort food.`

			# Scene change: the meeting with the Elders begins.
			label meeting
			`	After a short hour of frantic work, Teeneep leads you to a room within the government complex. Sayari explains, "This is a Special Adjunctive Triad: three Elders with diverse views, specialized knowledge, and the power to take limited immediate actions."` 

> 			`	The three Hai Elders are sat behind a table, and the Elder in the center gestures for all of you to sit down in those strange chairs. After a small, calm exchange in Hai between Sayari and the central Elder, he motions for you to speak.`

Reviewer: ```suggestion
			`	The three Hai Elders are sat behind a table, and the Elder in the center gestures for all of you to sit down in those strange Hai chairs. After a small, calm exchange in Hai between Sayari and the central Elder, he motions for you to speak.`
```
Forgot to really make sure the chairs are uncomfortable for you.

				goto "elder questions"
			label "elders secret"
			`	The middle Elder counters, "We cannot be certain that the pirate activity will continue. With the proper trades, we may be able to return to the status quo."`
			`	"That is a best-case scenario," says the right Elder, "but if matters continue to worsen, we'll change our approach." The other two nod in agreement.`
				goto diplomats
			label "Katya warning"
			`	The Elder on the right disagrees. "At this point, the Republic is too weakened to be a threat. I'm confident they will see reason, for now."`
			`	Fortunately, the middle Elder contradicts him. "We don't want to risk larger problems with humanity. If the situation continues to worsen, then we'll change our approach." The other two find that argument convincing and nod.`
			label diplomats
			`	An Elder concludes the meeting. "We have a plan of action on the pirate deals. However, I've heard enough of the 'kidnapping' conspiracy theory, so that ends this meeting." Teeneep is visibly furious, but remains silent as the three exit.`
			choice
				`	"That could have gone better."`
				`	"It sounds like we've made some progress!"`

> 			`	As you walk back to diplomatic areas, Sayari begins. "They didn't do anything, but at least they agreed to solve the problems when they become disasters."`

Reviewer: ```suggestion
			`	As you walk back to the diplomatic quarter, Sayari begins. "They didn't do anything, but at least they agreed to solve the problems when they become disasters."`
```

			`	Teeneep objects, "We can't wait for a disaster before solving this!"`
			`	Sayari agrees. "You're right, of course, in this matter. The Elders have ever struggled with change. They're not wise for this, but they were not elected to be so. They were elected to continue the long march of incremental development. Besides, you know far better than most exactly what challenges they contend with these past couple decades. More of them have died or retired than been unelected."`
			choice
				`	"What can we do to solve this, Sayari?"`
				`	"Teeneep, I think we need quick action. You mentioned something back in the cave."`

> 			`	Before she can respond, Xilin arrives with Batu in tow. They consult Sayari (the actual diplomat) and occasionally Teeneep. After a long dialog about the Navy knowing of the pirate troubles and willfully not intervening, Xilin sympathizes, suggesting the Hai ask the Navy to rescue those captured.`

Reviewer: This feels weird, but I'm not sure how to fix it yet.

Author: I think "(the actual diplomat)" is the narrator is expressing his dislike of Teeneep.  I shall rewrite that paragraph again.

Author: I have tweaked the paragraph. The narrator no longer expresses a dislike of Teeneep.

			`	When Teeneep makes a noise as if to speak, Sayari silences her with a gentle touch and thanks Xilin in the formal capacity. He briefly thanks you for delivering Batu safely and then the two excuse themselves to see if they can get a hold of someone before their day ended.`
			choice
				`	"Should we wait for Parliament to respond?"`
				`	"Do you have any other ideas?"`
					goto "other ideas"
			`	Teeneep sighs in exasperation. "We can't wait for your parliament to spend months deliberating. Sayari, you have a crisis of your own here, and I cannot accept a resolution without an avenue to rescue our people and put a stop to this for good. I would hope you cannot either..."`
			label "other ideas"
			`	Teeneep says with certainty, "I must go to the Unfettered. If they can be swayed then the Elders may be pressured into acting."`
			`	"Captain <last>," she says, giving you a calculating look. "I would very much appreciate it if you'd be so kind as to take me."`
			choice
				`	"Aren't those Hai pirates? No, that sounds unwise."`
				`	"Of course, Teeneep. Let's go."`

> 			`	"Teeneep, no! You cannot just commandeer the Captain to act as your chauffeur wherever you want to go," says Sayari. "<last> has an important role here, in this crisis."`

Reviewer: ```suggestion
			`	"Teeneep, no! You cannot just commandeer the Captain to act as your chauffeur wherever you want to go," says Sayari. "<last> has an important role here in this crisis."`
```
I hope that wasn't one of the commas I put in.

						has "Hai Reveal [A09] History of Appeasement: label deals"
						not "Hai Reveal [A09] History of Appeasement: label fix"
				# Optional follow-up questions are next, so you'll get the critical
				# information first.
				`	"Are the two of you usually this argumentative?"`
					goto friends
					to display
						has "Hai Reveal [A09] History of Appeasement: interrupted"
						not "Hai Reveal [A09] History of Appeasement: label friends"
				`	"Did all of the deals involve arming pirates?"`
					goto types
					to display
						has "Hai Reveal [A09] History of Appeasement: label deals"
						not "Hai Reveal [A09] History of Appeasement: label types"
				`	"Does Hai media not deal with this sort of thing?"`
					goto media
					to display
						has "Hai Reveal [A09] History of Appeasement: label fix"
						not "Hai Reveal [A09] History of Appeasement: label media"
				# End discussion.
				`	"We've discussed enough. Let's get to work."`
					goto go
					to display
						has "Hai Reveal [A09] History of Appeasement: label fix"

			# First phase of conversation: listen in or interrupt.
			label listen
			action
				set "Hai Reveal [A09] History of Appeasement: label listen"

> 			`	It is difficult to hear precisely without making yourself known, so you struggle to understand. At first, Teeneep is annoyed, maybe saying words like: pirate, victim, orchestra, and betrayal. Sayari's responses revolve around: necessity, an angry child, guilt, and eggs. Something sets off Teeneep, and she keeps screaming "tweezers." Sayari puts a paw gently on Teeneep's shoulders and says in a soothing maternal tone, "green the moon blue." At this point, you realize the conversation is entirely in Hai, which you don't speak, and you have no idea what the two women are actually discussing.`

Reviewer 3: @warp-core this is the specific bit I want you to look at

Author: Originally, this paragraph was written, so Captain Last knew at the beginning it was in Hai and was trying (and failing) to translate. Someone's suggestion (@EjoThims I think) changed it to not knowing it is in Hai until the end. Either makes sense to me, so if you want to interject on what is best, please do.

Reviewer 4: I'm not really sure why Grey wanted me to look at this specifically. It seems fine, I guess? Though, it's a bit weird that it doesn't occur to the character that they have no reason to be speaking the human language here sooner? It is fun to read, though.
Also, it should probably be `shoulder`, not `shoulders`, if Sayari is only using one paw, unless it's like a really big one.

Author: > they have no reason to be speaking the human language here sooner?

It made a lot more sense when Captain Last knew from the beginning that it was in Hai and was trying to translate. That way, though, you get the punchline at the beginning and it isn't as funny.

Reviewer 4: I think, overall, the "fun factor" outweighs the concern, I would change it slightly, though.
```suggestion
			`	It is difficult to hear precisely without making yourself known, so you struggle to understand. At first, Teeneep is annoyed, maybe saying words like: pirate, victim, orchestra, and betrayal. Sayari's responses revolve around: necessity, an angry child, guilt, and eggs. Something sets off Teeneep, and she keeps screaming "tweezers." Sayari puts a paw gently on Teeneep's shoulders and says in a soothing maternal tone, "green the moon blue." At this point, it becomes clear that the conversation is entirely in Hai, which you don't speak, and you have no idea what the two women are actually discussing.`
```

Reviewer 3: That's what I wanted to check, thank you warp

			`	Xilin frowns. "We should definitely present that case. They'll have to do another whole round of private briefings, and then no doubt a large debate. Even if we started today, Parliament won't be able to act for months. I'll kick things off, but you'll have to be patient."`
			`	When Teeneep makes a noise as if to speak, Sayari silences her with a gentle touch and thanks Xilin in the formal capacity. He briefly thanks you for delivering Batu safely and then the two excuse themselves to see if they can get a hold of someone before their day ended.`
			choice
				`	"Should we wait for Parliament to respond?"`
				`	"Do you have any other ideas?"`
					goto "other ideas"
			`	Teeneep sighs in exasperation. "We can't wait for your parliament to spend months deliberating. Sayari, you have a crisis of your own here, and I cannot accept a resolution without an avenue to rescue our people and put a stop to this for good. I would hope you cannot either..."`
			label "other ideas"

> 			`	Teeneep says with certainty, "I must go to the Unfettered. If they can be swayed then the Elders may be pressured into acting."`

Reviewer: ```suggestion
			`	Teeneep says with certainty, "I must go to the Unfettered. If they can be swayed, then the Elders may be pressured into acting."`
```

						has "Hai Reveal [A09] History of Appeasement: label deals"
						not "Hai Reveal [A09] History of Appeasement: label fix"
				# Optional follow-up questions are next, so you'll get the critical
				# information first.
				`	"Are the two of you usually this argumentative?"`
					goto friends
					to display
						has "Hai Reveal [A09] History of Appeasement: interrupted"
						not "Hai Reveal [A09] History of Appeasement: label friends"
				`	"Did all of the deals involve arming pirates?"`
					goto types
					to display
						has "Hai Reveal [A09] History of Appeasement: label deals"
						not "Hai Reveal [A09] History of Appeasement: label types"
				`	"Does Hai media not deal with this sort of thing?"`
					goto media
					to display
						has "Hai Reveal [A09] History of Appeasement: label fix"
						not "Hai Reveal [A09] History of Appeasement: label media"
				# End discussion.
				`	"We've discussed enough. Let's get to work."`
					goto go
					to display
						has "Hai Reveal [A09] History of Appeasement: label fix"

			# First phase of conversation: listen in or interrupt.
			label listen
			action
				set "Hai Reveal [A09] History of Appeasement: label listen"

> 			`	It is difficult to hear precisely without making yourself known, so you struggle to understand. At first, Teeneep is annoyed, maybe saying words like: pirate, victim, orchestra, and betrayal. Sayari's responses revolve around: necessity, an angry child, guilt, and eggs. Something sets off Teeneep, and she keeps screaming "tweezers." Sayari puts a paw gently on Teeneep's shoulders and says in a soothing maternal tone, "green the moon blue." At this point, it becomes clear that the conversation is entirely in Hai, which you don't speak, and you have no idea what the two women are actually discussing.`

Reviewer 3: ```suggestion
			`	It is difficult to hear precisely without making yourself known, so you struggle to understand. At first, Teeneep is annoyed, maybe saying words like: pirate, victim, orchestra, and betrayal. Sayari's responses revolve around: necessity, an angry child, guilt, and eggs. Something sets off Teeneep, and she keeps screaming "tweezers." Sayari puts a paw gently on Teeneep's shoulder and says in a soothing maternal tone, "green the moon blue." At this point, it becomes clear that the conversation is entirely in Hai, which you don't speak, and you have no idea what the two women are actually discussing.`
```

			action
				set "Hai Reveal [A09] History of Appeasement: interrupted"
			`	Realizing that you've arrived, Sayari tilts her head slightly to one side to indicate she's speaking to you, while not taking her eyes off Teeneep. "Welcome back, Captain <last>. I must apologize for my friend's state. She's just discovered some rather unfortunate information."`
				goto arrived
			label arrived
			`	"Unfortunate!" Teeneep very nearly shrieks the word. "They've been doing this for decades! Decades, Sayari! Tracking down and ultimately paying off pirates with our technology to leave us alone, even when they've hurt, killed, kidnapped, and enslaved our people!`
			`	"You knew! This whole time, for a half century, you knew, and you did nothing!"`
			`	Teeneep closes her eyes and breathes deeply until she has calmed herself down. It looks like this is a technique she's practiced many times.`
			`	The three of you sit down once Teeneep looks ready to talk.`
				goto questions
			label friends
			action
				set "Hai Reveal [A09] History of Appeasement: label friends"
			`	Sayari looks at you apologetically. "She's always been quite a spark, but we usually get along very well."`

> 			`	Teeneep adds, "Sayari has been my close friend and mentor since I was a head shorter than her."`

Reviewer 3: This order feels more appropriate to me.
```suggestion
			`	Teeneep adds, "Sayari has been my mentor and ultimately a close friend since I was a head shorter than her."`
```

Author: You're mixing past tense and present tense in a way that is quite confusing. I changed it to, "Sayari has been my mentor since I was a head shorter than her. I consider her my closest and most trusted friend."

				goto questions

			# Second phase of conversation: find out about the pirates.
			label deals
			action
				set "Hai Reveal [A09] History of Appeasement: label deals"
			`	Sayari says, "The Elders have been making deals with your pirates for a long time, in return for secrecy or ceasing mischief. We give them credits, supplies for their colonies, or weapons. I'm not privy to the details, but I've known for quite a while."`
			`	An enraged Teeneep counters. "They give them our weapons so that they can more efficiently go and enslave other people, rather than just dealing with them properly the way they should have! One hundred and seventy three reports, Sayari. That's how many times they've done this, or something like it, and those are just the ones that came up in the sweep. There's no telling how many others might be filed improperly, missing key words, or just outside of my search range.`
			`	"I thought you were a model of morality, Sayari. Now I know you're no better than they are. No better than... than..."`

> 			`	Sayari finishes Teeneep's thought. "No better than you? The curse of power is to sacrifice your morals for protection of people under your care. I think you know that well, Teeneep. Perhaps better than I?" Teeneep looks briefly ashamed when Sayari says that.`

Reviewer 3: Oof that cuts deep, really brings out the mentor in her, I like it.

Author: It is fair, though. Teeneep has her supporters kill Captain Last if you refuse to help her. She has to, of course, to keep the operation secret, but she's clearly no stranger to sacrificing her morals for the greater good.

			`	Teeneep takes a deep breath and accepts the plan with sudden and surprising grace. She gathers her pads, then sets up where indicated. "Alright, just get me some comfort food while I work on this."`
			`	Sayari asks you, "Her favorite foods are acorn curry and nutmeg pie. Which should we order?"`
			choice
				`	"Acorn curry sounds... good?"`
					goto curry
				`	"I'd love some nutmeg pie."`
			`	The pie tastes and feels like a meaty shepherd's pie, even though it's all nuts and vegetables. The subtle taste of nutmeg really brings it all together, accenting the acorns quite well.`
				goto meeting
			label curry
			`	You eat the most delicious curry you've ever tasted, with a perfect combination of acorns and the best Indian spices. It's amazing what Hai can do with nuts; no wonder Teeneep considers this a comfort food.`

			# Scene change: the meeting with the Elders begins.
			label meeting

> 			`	After a short hour of frantic work, Teeneep leads you to a room within the government complex. Sayari explains, "This is a Special Adjunctive Triad: three Elders with diverse views, specialized knowledge, and the power to take limited immediate actions."` 

Reviewer 3: Earlier on we imply that the elder council is somewhat large, here we have an extremely small selection. I would suggest making this either 5 or 7, though with a larger number perhaps 3 have some reserve power.

Author: This is not the Elder Council. It is only a few Elders the greater council chose to make a quick decision on a matter. The larger the group, the slower they'll take to get to a consensus. Three is the smallest group size that won't result in a tie.

I'm not clear on if you think this is the entire Elder Council, or if you're just asking for a larger group.

Author: I have added a dialog option for Sayari to expound on the Special Adjunctive Triad. It also gives the lore justification for Teeneep's "special Elder powers" when she was out in the field.

"Our governmental structure is directed towards stability, but sometimes there's a problem that merits rapid action. When needed, the Council will nominate a group of Elders, or even just one, to take actions the full Council would otherwise spend weeks deliberating upon. Their power is limited, and is defined when they're nominated, but their dictate is immediate. If necessity pushes beyond their preset limits, they can make a recommendation with a fast track to a full Council discussion. In trying times like these, they may get a meeting by tomorrow."

Reviewer 3: My thinking was that 3 is an extremely small number for a serious deliberation, but with the additional justification I suppose that does work somewhat neatly.

			`	The left Elder says, "Secrecy, mostly, to prevent a refugee crisis if the Hai become common knowledge. The bulk of the resources go to a few large pirate groups that control the anarchist worlds. We fund the development and protection of their planets, in return for them ensuring the wrong people don't find us."`
			`	Another Elder adds, "Most of the deals are much smaller, though: when a pirate gang causes mischief, we pay them to stop. Hostages, theft, raids, and the like. Usually, the same gang never bothers us again." Teeneep perks up at the word "hostages," but doesn't speak.`
				goto "elder questions"
			label "elders secret"
			`	The middle Elder counters, "We cannot be certain that the pirate activity will continue. With the proper trades, we may be able to return to the status quo."`
			`	"That is a best-case scenario," says the right Elder, "but if matters continue to worsen, we'll change our approach." The other two nod in agreement.`
				goto diplomats
			label "Katya warning"
			`	The Elder on the right disagrees. "At this point, the Republic is too weakened to be a threat. I'm confident they will see reason, for now."`

> 			`	Fortunately, the middle Elder contradicts him. "We don't want to risk larger problems with humanity. If the situation continues to worsen, then we'll change our approach." The other two find that argument convincing and nod.`

Reviewer 3: This problem is a less clear-cut one given the complexities of the Republic, and so I would expect that it'd be a debate on the council floor rather than the straight "yeah we can switch up how we deal with pirates".
```suggestion
			`	Fortunately, the middle Elder contradicts him. "We don't want to risk larger problems with humanity. If the situation continues to worsen, then we'll revisit that particular concern in session." The other two find that argument convincing and nod.`
```

Author: The Elders in this group were given the power to make immediate decisions on some matters, including this one.  I wanted this to sound like a definite decision (albeit delayed and conditional), not the usual Hai Elder indecisiveness.

Author: I updated it so they explain a few lines later, "The Elder Council asked us to take charge of the pirate situation, and we now have a plan of action on the pirate deals."

			label diplomats
			`	An Elder concludes the meeting. "We have a plan of action on the pirate deals. However, I've heard enough of this 'kidnapping' conspiracy theory. Our meeting is adjourned." Teeneep's fury is palpable, but she remains silent as the three exit.`
			choice
				`	"That could have gone better."`
				`	"It sounds like we've made some progress!"`
			`	As you walk back to the diplomatic quarter, Sayari begins. "They didn't do anything, but at least they agreed to solve the problems when they become disasters."`
			`	Teeneep objects, "We can't wait for a disaster before solving this!"`
			`	Sayari agrees. "You're right, of course, in this matter. The Elders have ever struggled with change. They're not wise for this, but they were not elected to be so. They were elected to continue the long march of incremental development. Besides, you know far better than most exactly what challenges they contend with these past couple decades. More of them have died or retired than been unelected."`
			choice
				`	"What can we do to solve this, Sayari?"`
				`	"Teeneep, I think we need quick action. You mentioned something back in the cave."`

> 			`	Before she can respond, Xilin arrives with Batu in tow. They consult Sayari, and occasionally Teeneep, in a long discussion about the Navy knowing of the pirate troubles and willfully not intervening. Xilin sympathizes and suggests the Hai ask the Navy to rescue those captured.`

Reviewer 3: Since it's established you're walking back to the diplomatic quarter just before this, I think it would feel more natural if you walked past where Xilin and Batu already were and they were to pull you into a room for this conversation.

Author: I just made that change.

> 			`	Batu cuts in. "I'm sorry, deputy minister, I have not had time to fully brief you. The Navy already considered this and said 'not under any circumstances' will they 'clean up the Hai's mess'. It's possible that five thousand missing civilians was not one of the things they considered when they said that."`

Reviewer 3: This felt too rushed how it sits in things now, he basically drops a bombshell and then rushes on. Establishing an explicit pause feels better here.
```suggestion
			`	Batu cuts in. "I'm sorry, deputy minister, I have not had time to fully brief you. The Navy already considered this and said 'not under any circumstances' will they 'clean up the Hai's mess'." In the slightly stunned pause following his words he draws breath and adds somewhat less intensely, "It's, uh, possible that five thousand missing civilians was not one of the things they considered when they said that."`
```

			`	When Teeneep makes a noise as if to speak, Sayari silences her with a gentle touch and thanks Xilin in the formal capacity. He briefly thanks you for delivering Batu safely and then the two excuse themselves to see if they can get a hold of someone before their day ended.`
			choice
				`	"Should we wait for Parliament to respond?"`
				`	"Do you have any other ideas?"`
					goto "other ideas"
			`	Teeneep sighs in exasperation. "We can't wait for your parliament to spend months deliberating. Sayari, you have a crisis of your own here, and I cannot accept a resolution without an avenue to rescue our people and put a stop to this for good. I would hope you cannot either..."`
			label "other ideas"
			`	Teeneep says with certainty, "I must go to the Unfettered. If they can be swayed then the Elders may be pressured into acting."`
			`	"Captain <last>," she says, giving you a calculating look. "I would very much appreciate it if you'd be so kind as to take me."`

> 			choice
> 				`	"Aren't those Hai pirates? No, that sounds unwise."`
> 				`	"Of course, Teeneep. Let's go."`

Reviewer 3: Side note; these options should be revisited at some point based on how much you've had to do with the Unfettered.
```suggestion
# TODO: These options should be revisited at some point based on how much you've had to do with the Unfettered, despite being only flavor.
			choice
				`	"Aren't those Hai pirates? No, that sounds unwise."`
				`	"Of course, Teeneep. Let's go."`
```

			`	"Teeneep, no! You cannot just commandeer the Captain to act as your chauffeur wherever you want to go," says Sayari. "<last> has an important role here in this crisis."`
			`	"And what would you have them do next then?" When Sayari seems at a loss to respond, Teeneep continues. "Exactly. Your crisis is mostly resolved behind closed doors, but this crisis requires transportation. The thing you seem to be missing is that they're the same crisis, Sayari. They can't be resolved separately."`

> 			`	Sayari sits with a huff, for a moment looking as old as she says she is. Teeneep waits patiently while her friend spends a good thirty seconds seriously thinking about something. "She is right, <last>," says Sayari eventually. "This is worth trying. Please take her to the Unfettered if that is what she wishes; their government is on <destination>, but please, be careful."`

Reviewer 3: If we change it so that Xilin and Batu pulled you into a meeting room of some kind I would reverse this change.

			choice
				`	"What can we do to solve this, Sayari?"`
				`	"Teeneep, I think we need quick action. You mentioned something back in the cave."`

> 			`	Before she can respond, Xilin arrives with Batu in tow. They consult Sayari, and occasionally Teeneep, who convey a long dialog about the Navy knowing of the pirate troubles and willfully not intervening. Xilin sympathizes and suggests the Hai ask the Navy to rescue those captured.`

Reviewer: ```suggestion
			`	Before she can respond, Xilin arrives with Batu in tow. They consult Sayari, and occasionally Teeneep, in a long discussion about the Navy knowing of the pirate troubles and willfully not intervening. Xilin sympathizes and suggests the Hai ask the Navy to rescue those captured.`
```
Sorry, this is better.

> 			`	Before she can respond, Xilin arrives with Batu in tow. They consult Sayari and occasionally Teeneep who convey a long dialog about the Navy knowing of the pirate troubles and willfully not intervening. Xilin sympathizes and suggests the Hai ask the Navy to rescue those captured.`

Reviewer: ```suggestion
			`	Before she can respond, Xilin arrives with Batu in tow. They consult Sayari, and occasionally Teeneep, who convey a long dialog about the Navy knowing of the pirate troubles and willfully not intervening. Xilin sympathizes and suggests the Hai ask the Navy to rescue those captured.`
```
